gguf-py/gguf/gguf.py

```diff
@@ -332,6 +357,17 @@ class TensorNameMap:
     block_mappings_cfg: dict[MODEL_TENSOR, tuple[str, ...]] = {
         # Attention norm
         MODEL_TENSOR.ATTN_NORM: (
+<<<<<<< HEAD
```

Comment:
Looks like you left some merge conflicts unresolved.

Reply:
Apologies--thanks for catching